jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnector.java

```diff
@@ -298,6 +301,26 @@ public int getAcceptors()
         return _acceptors.length;
     }
 
+    /**
+     * @return Returns the optional local name override
+     */
+    @ManagedAttribute("local name override")
+    public String getLocalNameOverride()
```

Comment:
I think just `getLocalName()` is enough and consistent with all the other APIs of the same name in `HttpChannel` and `Request`.